src/Summary/SummaryFactory.php

```diff
@@ -19,4 +19,26 @@ public function createFromAdherent(Adherent $adherent): Summary
     {
         return Summary::createFromMember($adherent, $this->slugger->slugify($adherent->getFullName()));
     }
+
+    public function createFromArray(array $data): Summary
```

Comment:
Please add a comment to explicit that this method should be used for test purpose only.